data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -44,7 +44,7 @@
         controller=ai
         recruit=Orcish Warrior,Goblin Knight
         {GOLD 100 150 200}
-        {INCOME 20 25 30}
+        {INCOME 10 20 30}
```

Comment:
This income variation is a little too much. I'm okay with something like {16 24 30} but having it be a third on easy difficulty means there'll be absolutely no pressure on you this scenario. Same applies for the other sides here.

Reply:
changed in 33a48a5